config/sidekiq_schedule.yml

```diff
@@ -2,3 +2,7 @@ queue_size_metric:
   cron: "*/30 * * * * *"
   class: "QueueSizeMetricJob"
   queue: default
+monthly_tasks_generation:
+  cron: '0 2 1 * *'
```

Comment:
Noob question: does cron cope with variable formats? The previous task uses 6 arguments, this one only five.

Reply:
The 6 argument version is a `sidekiq-cron` specific thing – it allows jobs to be scheduled with second-precision. See [this comment](https://github.com/ondrejbartas/sidekiq-cron/issues/67#issuecomment-148314162)

Reply:
Cool! The explanation I found gave the last param as the year, this is clearer now.